tasks/internal/request-and-upload-signature/request-and-upload-signature.yaml

```diff
@@ -18,16 +18,17 @@ spec:
   params:
     - description: A docker image of operator-pipeline-images for the steps to run in.
       name: pipeline_image
-      default: "quay.io/redhat-isv/operator-pipelines-images:released"
+      default: "quay.io/jluza/release-service-utils:latest"
```

Comment:
This needs to be changed when https://github.com/konflux-ci/release-service-utils/pull/365 is merged and released